common/app/templates/inlineJS/nonBlocking/prepareCmp.scala.js

```diff
@@ -59,7 +59,7 @@ try {
                     storeConsentGlobally: false,
                     storePublisherData: false,
                     logging: false,
-                    gdprApplies: false,
```

Comment:
A suggestion, if we no longer have a `false` case, maybe we can remove completely this field?

Reply:
We need to keep it because the TCF IAB implementation requires this parameter. So even thought in our case it's never set to false we still need to have it on the Object.